Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Internal_-_Generic_v2.yml

```diff
@@ -310,14 +332,35 @@ tasks:
       {
         "position": {
           "x": 170,
-          "y": 570
+          "y": 510
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    continueonerrortype: ""
+    fieldMapping:
+    - incidentfield: Detected Internal IPs
```

Comment:
This field is associated only with the PANW Endpoint Malware type.
Maybe we should change that accordingly?
or use extended context?
![image](https://github.com/demisto/content/assets/88268646/272937b6-0a04-4835-bd3a-50b1a42f9990)

Reply:
I'll remove it, just thought it'd be nice